cmd/soci/commands/push.go

```diff
@@ -38,6 +39,11 @@ import (
 	"oras.land/oras-go/v2/registry/remote/auth"
 )
 
+const (
+	maxConcurrentUploadsFlag    = "max-concurrent-uploads"
+	defaultMaxConcurrentUploads = 10
```

Comment:
ORAS already has a default value of 3, I think we should be more in-line with it. A value of 3 or 0 are both ok. I think the latter may be better with a comment explaining that this will set the value to the default one set in ORAS.

Reply:
Sure pushed a default value of 3. 0 is a little strange IMO for default behavior to rely on third party library. Also updated docs with the default value change.

Reply:
Sure, I suggested zero to ensure we stay consistent but deviating in this case is probably not a problem.